src/components/parent/ScoreCard.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { MouseEventHandler } from "react";
+// import { IonAlert, IonButton, IonIcon } from "@ionic/react";
+import "./ScoreCard.css";
+import { OverlayEventDetail } from "@ionic/react/dist/types/components/react-component-lib/interfaces";
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+} from "@mui/material";
+import { t } from "i18next";
+import { chevronForward } from "ionicons/icons";
+import { title } from "process";
+import { IoCloseCircle } from "react-icons/io5";
+import { PAGES } from "../../common/constants";
+import ScoreCard from "../LessonCardStarIcons";
+import StarImages from "./ScoreCardStarIcons";
+import { AiTwotoneHeart } from "react-icons/ai";
+import { GrFormNext } from "react-icons/gr";
+
+const NewScoreCard: React.FC<{
```

Comment:
where are you using this NewScoreCard.
if you are using it somewhere change the name to ScoreCard